core/src/main/java/io/opencensus/stats/StatsManager.java

```diff
@@ -32,5 +32,5 @@
   /**
    * Returns the default {@link StatsContextFactory}.
    */
-  abstract StatsContextFactory getStatsContextFactory();
+  public abstract StatsContextFactory getStatsContextFactory();
```

Comment:
I only exposed this method to make the example in this PR work.  We are redesigning the StatsContext class so that it is contained in the API library and doesn't have to be so tightly coupled to the StatsManager (#323).  StatsManager will still have "record" methods, though:

explicit context: `void record(TagContext tags, MeasureMap values)`
implicit context: `void record(MeasureMap values)`

Do you think that registering/querying views and recording stats should also be separate?  I'm not sure how we can separate those parts, since they need to work with shared state.  If people inject them separately, the objects would still need to come from the same StatsManagerFactory instance (or some root object with a better name) that can ensure that the objects are using the same state.

One idea is to call the singleton StatsComponent and give it methods getViewManager() and getStatsRecorder().  Then you could get the ViewManager and StatsRecorder from the singleton and pass them around separately.  Would that address the problem with bundling unrelated functionalities?